opendj-server-legacy/src/main/java/org/opends/server/util/Platform.java

```diff
@@ -324,12 +204,11 @@ else if (ks.containsAlias(alias))
           throw new KeyStoreException(msg.toString());
         }
 
-        final Object keypair = newKeyPair(keyType);
-        final Object subject = newX500Name(dn);
-        generate(keypair, keyType.keySize);
-        final PrivateKey privateKey = getPrivateKey(keypair);
-        final Certificate[] certificateChain = new Certificate[] {
-          getSelfCertificate(keypair, subject, validity * SEC_IN_DAY)
+        var keyPair = newKeyPair(keyType);
```

Comment:
Not a fan of `var` here. It is a vaild construct when you have explicit constructor call on the right side, but less valid when it is a method call.